components/automate-ui/src/app/entities/notification_rules/notification_rule.effects.ts

```diff
@@ -45,7 +54,63 @@ export class NotificationRuleEffects {
       const msg = payload.error.error;
       return new CreateNotification({
         type: Type.error,
-        message: `Could not get notifications: ${msg}`
+        message: `Could not get notifications: ${msg || payload.error}`
+      });
+    }));
+
+  @Effect()
+  createNotificationRule$ = this.actions$.pipe(
+    ofType(NotificationRuleActionTypes.CREATE),
+    mergeMap(({ payload, username, password }: CreateNotificationRule) =>
+    this.requests.createNotificationRule( payload, username, password ).pipe(
+      map((resp) => new CreateNotificationRuleSuccess(NotificationRule.fromResponse(resp.rule))),
+      catchError((error: HttpErrorResponse) =>
+        observableOf(new CreateNotificationRuleFailure(error))))));
+
+  @Effect()
+  createNotificationRuleSuccess$ = this.actions$.pipe(
+    ofType(NotificationRuleActionTypes.CREATE_SUCCESS),
+    map(({ payload  }: CreateNotificationRuleSuccess) => new CreateNotification({
+    type: Type.info,
+    message: `Created notification ${payload.name}.`
+  })));
+
+  @Effect()
+  createNotificationRuleFailure$ = this.actions$.pipe(
+    ofType(NotificationRuleActionTypes.CREATE_FAILURE),
+    filter(({ payload }: CreateNotificationRuleFailure) => payload.status !== HttpStatus.CONFLICT),
+    map(({ payload }: CreateNotificationRuleFailure) => new CreateNotification({
+        type: Type.error,
+        message: `Could not create notification: ${payload.error.error || payload}.`
+      })));
+
+  @Effect()
+  deleteNotificationRule$ = this.actions$.pipe(
+    ofType(NotificationRuleActionTypes.DELETE),
+    mergeMap(({ payload: { id, name } }: DeleteNotificationRule) =>
+      this.requests.deleteNotificationRule(id).pipe(
+        map(() => new DeleteNotificationRuleSuccess({id, name})),
+        catchError((error: HttpErrorResponse) =>
+          observableOf(new DeleteNotificationRuleFailure(error))))));
+
+  @Effect()
+  deleteNotificationRuleSuccess$ = this.actions$.pipe(
+    ofType(NotificationRuleActionTypes.DELETE_SUCCESS),
+    map(({ payload: { name } }: DeleteNotificationRuleSuccess) => {
+      return new CreateNotification({
+        type: Type.info,
+        message: `Deleted rule ${name}.`
+      });
+    }));
+
+  @Effect()
+  deleteNotificationRuleFailure$ = this.actions$.pipe(
+    ofType(NotificationRuleActionTypes.DELETE_FAILURE),
+    map(({ payload }: DeleteNotificationRuleFailure) => {
+      const msg = payload.error.error;
+      return new CreateNotification({
+        type: Type.error,
+        message: `Could not delete rule: ${msg || payload.error}`
```

Comment:
To be consistent 😉 let's use this (like L84):
```suggestion
        message: `Could not delete rule: ${payload.error.error || payload.error}`
```